packages/transaction-controller/package.json

```diff
@@ -70,7 +70,7 @@
     "@metamask/auto-changelog": "^3.4.4",
     "@metamask/eth-json-rpc-provider": "^4.1.1",
     "@metamask/ethjs-provider-http": "^0.3.0",
-    "@metamask/keyring-api": "^8.0.0",
+    "@metamask/keyring-api": "^8.0.1",
```

Comment:
For the @MetaMask/confirmations team: this is the only diff that affects the confirmations codeowner repos (aside from the `typescript` bumps and test build script changes in every package).